.git2gus/config.json

```
@@ -1,5 +1,5 @@
 {
   "productTag": "a1aB0000000g309IAA",
-  "defaultBuild": "234",
+  "defaultBuild": "236",
```

Should this be 238 now that we cut the `spring22` branch?

```suggestion
  "defaultBuild": "238",
```

---

How was this working before?

---

It was tagging the release version incorrectly. See for example https://github.com/salesforce/lwc/issues/2571#issuecomment-961479362